spec.html

```diff
@@ -8190,10 +8190,13 @@ <h1>[[Call]] ( _thisArgument_, _argumentsList_ )</h1>
       <p>The [[Call]] internal method of an ECMAScript function object _F_ takes arguments _thisArgument_ (an ECMAScript language value) and _argumentsList_ (a List of ECMAScript language values). It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _F_ is an ECMAScript function object.
-        1. If _F_.[[IsClassConstructor]] is *true*, throw a *TypeError* exception.
         1. Let _callerContext_ be the running execution context.
         1. Let _calleeContext_ be PrepareForOrdinaryCall(_F_, *undefined*).
         1. Assert: _calleeContext_ is now the running execution context.
+        1. If _F_.[[IsClassConstructor]] is *true*, then
+          1. Let _error_ be a newly created *TypeError* object.
+          1. Remove _calleeContext_ from the execution context stack and restore _callerContext_ as the running execution context.
+          1. Return AbruptCompletion(_error_).
```

Comment:
I believe this was about making the TypeError instance come from the realm in which the class was defined rather than the realm in which it was called (see [notes](https://github.com/tc39/notes/blob/master/meetings/2020-11/nov-17.md#default-constructors-and-spread-operations)).

Reply:
I see, that probably warrants a note so it doesn't get refactored away later.